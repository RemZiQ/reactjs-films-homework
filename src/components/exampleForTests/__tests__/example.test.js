import React from 'react';
import renderer from 'react-test-renderer';
import Example from '../example';


it('renders well', () => {
  const tree2 = renderer.create(<Example />).toJSON();
  expect(tree2).toMatchSnapshot();
});
