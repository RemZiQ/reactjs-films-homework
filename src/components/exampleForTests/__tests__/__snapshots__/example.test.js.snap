// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders well 1`] = `
<h2>
  Some text
</h2>
`;
